# LightControlStatus.msg - 灯光控制状态反馈
# 灯光控制状态
int8 enable             # 使能状态码
string enable_text      # 使能状态文本描述
int8 mode               # 灯光模式码
string mode_text        # 灯光模式文本描述